src/code/gfxprint.c

```diff
@@ -132,15 +132,13 @@ u8 sGfxPrintFontData[(16 * 256) / 2] = {
 };
 
 #define gDPSetPrimColorMod(pkt, m, l, rgba)                                                    \
-    _DW({                                                                                      \
```

Comment:
since we know now that this macro is only relevant in this file, im wondering if we want to give it a more custom name that doesnt suggest it was actually an addition to gbi. This is more of a question for everyone tho and doesnt need to be answered right now